# HG changeset patch
# User Carlo de Falco <kingcrimson@tiscali.it>
# Date 1298304745 -3600
#      Mon Feb 21 17:12:25 2011 +0100
# Node ID 46b3883d800c4db59b6286a0a21f93641d43bfe2
# Parent  2f83974d5a6d5f247de8d5b786d05faefa7c4c90
fixed typo in PCRE style RE

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,12 +1,16 @@
 2010-02-20  Karsten Trulsen  <karstent@math.uio.no>
 
 	* signal/fftshift.m: Fix broken function.  Bug 32442.
 
+2010-02-21  Carlo de Falco  <kingcrimson@tiscali.it>
+
+	* pkg/get_forge_pkg.m: Fix typo in new PCRE style regular expression.
+
 2010-02-19  Rik  <octave@nomad.inbox5.com>
 
 	* io/dlmwrite.m, pkg/get_forge_pkg.m, plot/__gnuplot_ginput__.m,
 	plot/__go_draw_axes__.m, testfun/runtests.m: Use PCRE regular
 	expressions to simplify scripts.
 
 2011-02-19  Ben Abbott  <bpabbott@mac.com>
 
diff --git a/scripts/pkg/get_forge_pkg.m b/scripts/pkg/get_forge_pkg.m
--- a/scripts/pkg/get_forge_pkg.m
+++ b/scripts/pkg/get_forge_pkg.m
@@ -38,17 +38,17 @@ function [ver, url] = get_forge_pkg (nam
   name = tolower (name);
 
   ## Try to download package's index page.
   [html, succ] = urlread (sprintf ("http://octave.sourceforge.net/%s/index.html", name));
   if (succ)
     ## Remove blanks for simpler matching.
     html(isspace(html)) = [];
     ## Good. Let's grep for the version.
-    pat = "<tdclass=""package_table"">PackageVersion:</td><td>([\d.]*)</td>";
+    pat = "<tdclass=""package_table"">PackageVersion:</td><td>([\\d.]*)</td>";
     t = regexp (html, pat, "tokens");
     if (isempty (t) || isempty(t{1}))
       error ("get_forge_pkg: could not read version number from package's page");
     else
       ver = t{1}{1};
       if (nargout > 1)
         # Build download string.
         urlbase = "http://downloads.sourceforge.net/octave/%s-%s.tar.gz?download";
